### PR TITLE
Initial content for MySQL

### LINKED ISSUE #78: MySQL overview
Add initial product content following existing templates/conventions

### LINKED ISSUE #87: MySQL getting started
Create a getting started guide for the product

### LINKED ISSUE #96: MySQL connecting article
Detail how to find and use the connection details for this service

### PR DESCRIPTION
**Pull request by lornajane**
# What changed, and why it matters

We don't have any MySQL content on the site yet! This PR adds the landing page, getting started content, and content from the original getting started page on help.aiven.io (includes a code sample and some client connection examples), plus the configuration options page.

Fixes #96. Fixes #87 . Fixes #78 .

I don't have MySQL Workbench installed so we must review the details and maybe update the screenshot for that page.

### CODE REVIEW

File: code/products/mysql/connect.py [hunk 1]
```diff
@@ -0,0 +1,25 @@
+import pymysql
+
+timeout = 10
+connection = pymysql.connect(
+
```
Comment (laysauchoa) [inline code/products/mysql/connect.py:5]: does it make sense to remove this extra line?

File: docs/products/mysql/get-started.rst [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+This view shows the connection parameters for your MySQL service and its
+current status. You can make changes to the service configuration here,
+even while the service is being built. You can find the available
+configuration options on the :doc:`reference page <reference/advanced_params>`.
+
+The "Status" indicator says " **REBUILDING** " while the service is
+being created. Once the service is up and running, the light changes to
+green and the indicator says " **RUNNING** ".
+
+Note that while typically services start in a couple of minutes, the
```
Comment (laysauchoa) [inline docs/products/mysql/get-started.rst:23]: should this be in a note instead? what do you think?

File: docs/products/mysql/howto/connect-with-python.rst [hunk 3]
```diff
@@ -0,0 +1,54 @@
...
+------------------      -------------------------------------------------------------
+``MYSQL_PASSWORD``      Password for this user
+==================      =============================================================
+
+Pre-requisites
+''''''''''''''
+
+For this example you will need:
+
+* Python 3.6 or later
```
Comment (laysauchoa) [inline docs/products/mysql/howto/connect-with-python.rst:28]: off-topic: Python 3.6 will reach end of life next week (23 Dec).

**Review by laysauchoa — APPROVED** [review]
looks really good!

File: code/products/mysql/connect.py (hunk 1 above)
Comment (lornajane) [reply code/products/mysql/connect.py]: Thanks, I did that

File: docs/products/mysql/get-started.rst (hunk 2 above)
Comment (lornajane) [reply docs/products/mysql/get-started.rst]: Good suggestion! Updated.

File: docs/products/mysql/howto/connect-with-python.rst (hunk 3 above)
Comment (lornajane) [reply docs/products/mysql/howto/connect-with-python.rst]: bumped to 3.7

**Comment by lornajane** [comment]
@laysauchoa if you approve a pull request, go ahead and merge it as well. If it is good to go, there is no reason to wait!

**Comment by laysauchoa** [comment]
> @laysauchoa if you approve a pull request, go ahead and merge it as well. If it is good to go, there is no reason to wait!

Got it! Good to go :)

**Pull request merged**